src/models/User.ts

```diff
@@ -0,0 +1,29 @@
+import {Model, DataTypes} from 'sequelize';
+
+import sequelize from '../db';
+
+class User extends Model {
+  public id!: number;
+  public email!: string;
+  public password!: string;
+}
+
+User.init(
+  {
+    userId: {
+      type: DataTypes.INTEGER,
```

Comment:
Make the `init` function and the migration portion the same

Reply:
ok

Reply:
Yea ps this one should be id too, my bad